apps/system-e2e/src/tests/islandis/application-system/acceptance/car-recycling.spec.ts

```diff
@@ -75,6 +75,16 @@ applicationTest.describe('Car recycling', () => {
             name: label(carRecyclingMessages.cars.selectedTitle),
           }),
         ).toBeVisible()
+
+        const mileage = page.getByRole('textbox', {
+          name: label(carRecyclingMessages.cars.mileage),
+        })
+
+        if (mileage) {
+          await mileage.selectText()
+          await mileage.type('12345')
+        }
```

Comment:
Why is this conditional? We should be using fake users, which we _know_ have mileage.

Reply:
You are right, my mistake. 
I have removed the if statement